metadata.json

```
@@ -16,6 +16,10 @@
     {
       "name": "puppetlabs/stdlib",
       "version_requirement": ">= 4.25.0 < 7.0.0"
+    },
+    {
+      "name": "camptocamp/systemd",
+      "version_requirement": ">= 2.5.1"
```

Please set the upper limit to 3.0.0:
```suggestion
      "version_requirement": ">= 2.5.1 < 3.0.0"
```

Is there a reason for setting the lower boundary to 2.5.1? If I introduce a new dependency I usually set the lower version boundary to the latest release because that's what we test against.